### DISCUSSION TITLE
Are Raspberry Pi 4 (w. Ubuntu 22.04) supported?

### DISCUSSION THREAD

**Discussion by joeH68** [emersion/maddy#528] [Q&A]
Hi guys, 

I found maddy while looking for a **thin intranet mailserver** to service legacy IMAP clients embedded in older devices. 

There will be no connection to the outside world (internet), thus no need for SSL, spam filter, firewall.

Ideally I would like to have it running in a headless RPI4 4GB server under Ubuntu (currently version 22.04).

My question: is this a supported configuration?  any experience with maddy in combination with RPI4 and/or Ubuntu?

Thanks!

**Comment by foxcpp** [comment]
maddy should run fine on RPi4. Default configuration will need to be adjusted to remove unused functionality. Feel free to ask any questions you have.

**Comment by joeH68** [comment]
Thanks for the prompt and positive response :)   I'm interested and will like to give it a try.

Is there anything like a "cookbook for dummies" I may use for installing maddy?  I'm afraid my Linux sysadmin knowhow is almost not existent. I can install (e.g. sudo apt-get ...) and configure the required conf files, but haven't used docker or compiled donwloaded code before.
